spec/resources/cdl/charge_manager_spec.rb

```diff
@@ -60,6 +60,20 @@ def item_ids(source_metadata_identifier:)
           reloaded_charges = Valkyrie.config.metadata_adapter.query_service.find_by(id: resource_charge_list.id)
           expect(reloaded_charges.charged_items).to be_present
         end
+        it "removes any existing holds for that netid" do
```

Comment:
These are existing holds for the same item_id, correct? If so can you update the spec title to: removes any existing holds for the same item_id for that netid

Reply:
No, holds aren't for a specific item_id, just the bib as a whole.

Reply:
oh ok then!